release-notes/opensearch-knn.release-notes-2.17.0.0.md

```diff
@@ -12,6 +12,7 @@ Compatible with OpenSearch 2.17.0
 ### Enhancements
 * Adds iterative graph build capability into a faiss index to improve the memory footprint during indexing and Integrates KNNVectorsFormat for native engines[#1950](https://github.com/opensearch-project/k-NN/pull/1950)
 * Add model version to model metadata and change model metadata reads to be from cluster metadata [#2005](https://github.com/opensearch-project/k-NN/pull/2005)
+* Enable training with quantization framework [#2100](https://github.com/opensearch-project/k-NN/pull/2100)
```

Comment:
remove this one because we havent released quant framework yet - this is more of experience fix